api-scripts/search/search_keywordSearch/search_keywordSearch.jmx

```diff
@@ -0,0 +1,358 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks good, just think can we enable reports? They are disabled now:
![image](https://user-images.githubusercontent.com/9094650/119794641-5f8d9c80-bee0-11eb-9f0f-3f5586d8dde3.png)

Reply:
yes, I guess so, but I saw that in inventory module they was disabled